test/compile_errors.zig

```diff
@@ -38,6 +38,74 @@ pub fn addCases(ctx: *Cases, b: *std.Build) !void {
         });
     }
 
+    {
+        const case = ctx.obj("identifier escapes", b.graph.host);
+
+        case.addError(
+            \\export fn a() void {
+            \\    return @"Á";
+            \\}
+            \\
+            \\export fn b() void {
+            \\    return @"B";
+            \\}
+            \\
+            \\export fn c() void {
+            \\    return @"a\nb";
+            \\}
+        , &[_][]const u8{
+            \\:2:12: error: use of undeclared identifier @"\xc3\x81"
+            ,
+            \\:6:12: error: use of undeclared identifier 'B'
+            ,
+            \\:10:12: error: use of undeclared identifier @"a\nb"
+        });
+    }
+
+    {
+        const case = ctx.obj("unmatched parentheses", b.graph.host);
+
+        case.addError(
+            \\export fn a() void {
+            \\}
+            \\}
+        , &[_][]const u8{
+            ":3:1: error: unmatched curly brace",
```

Comment:
Two error messages for one problem seems bad
It would be better if this was a note

Reply:
Most of the time, the unmatched brace is the underlying error and the particular parse error coming out of that is just a symptom, so I'd definitely expect the unmatched-error first, not as a note on something else.